documentation/src/docs/asciidoc/release-notes/release-notes-5.12.0-M1.adoc

```diff
@@ -83,7 +83,10 @@ JUnit repository on GitHub.
 [[release-notes-5.12.0-M1-junit-jupiter-bug-fixes]]
 ==== Bug Fixes
 
-* ❓
+* Provide _runtime_ enclosing types of `@Nested` test classes and contained test methods
+  to `DisplayNameGenerator` implementations. Prior to this change, such generators were
+  only able to access the enclosing class in which `@Nested` was declared, but they could
+  not access the concrete type of the enclosing instance.
```

Comment:
```suggestion
  not access the concrete runtime type of the enclosing instance.
```

Reply:
I realize it already says "type of the enclosing instance" (as opposed to "enclosing type"), but I think it makes it a bit more clear to developers that we're talking about the runtime type vs. the static type.